lib/remote-objects.js

```diff
@@ -132,7 +134,7 @@ RemoteObjects.prototype.connect = function(url, name) {
   }
 
   var Adapter = this.adapter(name);
-  var adapter = new Adapter(this);
+  var adapter = new Adapter(this, this.options);
```

Comment:
As I mentioned above, `this.options` are usually different from adapter options. For example (and IIRC), the configuration of REST adapter is specified in `rest` property of remoting options.

I think we should add `options` argument to `connect`, to make this consistent with `handler` method.

Reply:
OK I don't need this at all anymore, not even for the connect. As the Adapter gets it off the rest config.